pkg/system/phase4_configuring.go

```diff
@@ -36,6 +36,9 @@ func (r *Reconciler) ReconcilePhaseConfiguring() error {
 	if err := r.ReconcileSecretAdmin(); err != nil {
 		return err
 	}
+	if err := r.NBClient.RegisterToCluster(); err != nil {
```

Comment:
For now I would comment this call, because we don't know if this behaves nicely with many notifications and if this doesn't trigger too many reconciles...